docker/Makefile

```diff
@@ -136,14 +141,14 @@ TEST_FILES += -f services_base.yaml
 TEST_FILES += -f ccf_base.yaml
 TEST_FILES += -f test.yaml
 
-build_test : repository build_base build_services_base build_ccf_base
-	PDO_VERSION=${PDO_VERSION} PDO_USER_UID=$(PDO_USER_UID) PDO_GROUP_UID=$(PDO_GROUP_UID) \
-		docker-compose $(TEST_FILES) build \
-			--build-arg PDO_VERSION=${PDO_VERSION}
+build_test : repository build_services build_ccf build_client
+
+test : clean_config clean_repository test_no_reset
+
+test_no_reset : build_test stop_all
+	PDO_VERSION=$(PDO_VERSION) docker-compose $(TEST_FILES) up --abort-on-container-exit
+	PDO_VERSION=$(PDO_VERSION) docker-compose $(TEST_FILES) down
 
-test : clean_config clean_repository build_test
-	PDO_VERSION=${PDO_VERSION} docker-compose $(TEST_FILES) up --abort-on-container-exit
-	PDO_VERSION=${PDO_VERSION} docker-compose $(TEST_FILES) down
 
```

Comment:
this is lovely in theory... but not particularly useful in practice. if i'm re-running the tests its because i made changes to the repository. so... does this really add more value than the complexity it adds?

Reply:
it tremendously helps, e.g., if you change anything related to docker building which is _not_ taking from the repository (which of course arguably could be seen as an issue)

Reply:
but how often to you do this? that's a very small fraction of the time.